py/server/deephaven/experimental/iceberg.py

```diff
@@ -124,7 +203,13 @@ def tables(self, namespace: str) -> Table:
 
     def snapshots(self, table_identifier: str) -> Table:
         """
-        Returns information on the snapshots of the specified table as a Deephaven table.
+        Returns information on the snapshots of the specified table as a Deephaven table. The table contains the
+        following columns:
+        - `Id`: the snapshot identifier (can be used for updating the table or loading a specific snapshot).
+        - `TimestampMs`: the timestamp of the snapshot.
```

Comment:
Why are we returning what appears to be milliseconds?  Why not have the column as an `Instant`?  Either way, the doc does not indicate that it is milliseconds.

Reply:
This was corrected in https://github.com/deephaven/deephaven-core/pull/5865